src/rps-game.js

```diff
@@ -0,0 +1,106 @@
+//Import functions to reference
+import getThrow from './get-throw.js';
+
+//Reference needed DOM elements
+const playGame = document.getElementById('play-game');
+const generatedImage = document.getElementById('generated-hand');
+const userImage = document.getElementById('user-image');
+let outcomeMessage = document.getElementById('outcome-message');
+const resultsSection = document.getElementById('results-section');
+const winCount = document.getElementById('win-count');
+const lossCount = document.getElementById('loss-count');
+let userBet = document.getElementById('user-bet');
+const userBalance = document.getElementById('user-balance');
+
+//Initialization
+let wins = 0;
+let losses = 0;
+let balance = 500;
+
+//Event Handlers
+playGame.addEventListener('click', () => {
+    //run function to generate computer's throw
+    const computerThrow = getThrow();
+    
+    //get user choice from input
+    let userChoice = document.querySelector('input:checked').value;
+    let bet = userBet.value;
+    
+    //define the image src of the computer's throw
+    generatedImage.classList.remove('hidden');
+    const src = 'assets/computer-' + computerThrow + '.png';
+    generatedImage.src = src;
```

Comment:
you should remove the hidden class only after you have set the new source for the image file, things move so fast that the user will probably not see whatever the previous image was before the new source is set, but it is something to think about.